coldfront/plugins/qumulo/static/allocation.js

```diff
@@ -22,30 +22,43 @@ let confirmed = false;
 
 const submitButton = document.getElementById("allocation_form_submit");
 submitButton.addEventListener("click", (event) => {
-  const id_project_pk_elem = document.getElementById("div_id_project_pk");
-  const smb = protocols.find((protocol) => protocol.value === "smb");
+  const smb = protocols.find((protocol) => protocol.value === "smb"),
+        modalShouldDisplay = () => {
+          /*
```

Comment:
I think we don't need to include this comment.  I try to avoid and discourage any sort of extra commentary if we can imply it from the code, and if we can't imply it from the code, then the code needs refined.

In this case, the most I think we should have is something like `//detect if we on "Create Allocation form (vs Update)`